docs/irma-frontend.md

```diff
@@ -160,7 +160,7 @@ particular plugin on GitHub. There are links in the plugin overview [above](#ava
 For convenience we already bundled `irma-core`, `irma-web`, `irma-popup` and `irma-client` together with the default styling
 from `irma-css`. We also added polyfills in this package to realize support for Internet Explorer 11.
 The package can be installed from the npm registry.
-The bundled package can also be downloaded directly [here](https://gitlab.science.ru.nl/irma/github-mirrors/irma-frontend-packages/-/jobs/artifacts/master/raw/irma-frontend/dist/irma.js?job=irma-frontend).
+The bundled package can also be downloaded directly [here](http://github.com/privacybydesign/irma-frontend-packages/releases/latest/download/irma.js).
```

Comment:
A couple of the new links are http instead of https. could be changed to https for consistency

Reply:
Hmm weird, I copy-pasted the url from my web browser. I'll fix.